releasenotes/notes/hhlrefactoring-0b5048d0e6e8d9e0.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
My understanding was that the intent is to have a single larger release note around the Aqua migration and the algorithms & opflow that have been moved here to Terra. As such I think this reno file can be deleted

Reply:
deleted